sim/shaman/runes.go

```diff
@@ -146,7 +148,7 @@ func (shaman *Shaman) applyMaelstromWeapon() {
 	buffSpellId := 408505
 	buffDuration := time.Second * 30
 
-	ppm := 10.0
+	ppm := 15.0
```

Comment:
This is only true if the main hand is imbued with Windufury

Reply:
Good call out, forgot that piece. Implemented it similar to our Lava Lash Flametongue code.